applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -149,7 +166,12 @@ def ExecuteInitialize(self):
             mesh_dependent_constant = self.settings["hessian_strategy_parameters"]["mesh_dependent_constant"].GetDouble()
             if (mesh_dependent_constant == 0.0):
                 self.settings["hessian_strategy_parameters"]["mesh_dependent_constant"].SetDouble(0.5 * (self.dim/(self.dim + 1))**2.0)
-        
+        elif (self.strategy == "SPR"):
```

Comment:
what is an spr strategy?
Is this a solving_strategy?

Reply:
Superconvergent Patch Recovery